src/lib/components/pages/judge/index.tsx

```diff
@@ -0,0 +1,128 @@
+"use client";
+
+import { useState } from "react";
+
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "../../ui/dialog";
+
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "../../ui/form";
+import { Input } from "../../ui/input";
+import { Label } from "../../ui/label";
+import { Button } from "../../ui/button";
+
+interface IListTeam {
+  id: number;
+  name: string;
+  isValidate: boolean;
+}
+
+const CardTeam = (props: { name: string; isValidate: boolean }) => {
+  const { name, isValidate } = props;
+  return (
+    <div className="py-5 px-6 rounded-lg shadow-md border flex justify-between items-center">
+      <span>{name}</span>
```

Comment:
Better using tag heading or paragraph for better semantic, or you can use span but you must wrap the span with paragraph or heading tag